lib/helpers/formatters.js

```diff
@@ -0,0 +1,33 @@
+export const formatScope = (scope = '') => (scope ? `(${scope.trim()})` : scope.trim());
+
+export const combineTypeScope = (type, scope) => {
+  let thisType = type;
+
+  const thisScope = formatScope(scope);
+
+  // add scope correctly if ':' is at the end
+  if (thisScope.length > 0) {
+    if (thisType.charAt(thisType.length - 1) === ':') {
```

Comment:
this could probably become:

```js
  if (
    thisScope.length > 0
    && thisType.charAt(thisType.length -1) === ':'
  ) {
```

Reply:
Cannot really work as I have to remove the `:` just in this case.

I will always add the scope when `thisScope.length > 0`.